src/e2e-test/java/io/cdap/cdap/ui/utils/Helper.java

```diff
@@ -217,4 +271,42 @@ public static void setCdapTheme(String themePath) {
       Assert.fail(e.getMessage());
     }
   }
+
+  public static void takeScreenShot() {
```

Comment:
can take screenshots now for debugging locally. Maybe we can try to save error screenshots into the build debug files in github actions in the future

Reply:
Could we consolidate these two? https://github.com/cdapio/cdap-ui/blob/develop/src/e2e-test/java/io/cdap/cdap/ui/utils/Commands.java#L314

Reply:
FYI, you can add the reports/screenshots to the build debug files by giving the workflow file their corresponding directory here:
https://github.com/cdapio/cdap-ui/blob/da96db321b5e1621ab89481508265afa906889bf/.github/workflows/build.yml#L95